rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel    = "nightly-2022-11-02"
+channel    = "nightly-2022-09-28"
```

Comment:
What's the issue with `nightly-2022-11-02` on macos?

Reply:
This nightly version was not available on this `aarch64-apple-darwin` target.

Reply:
Maybe we can move to `2022-11-30`?

Reply:
I'm surprised that `nightly-2022-11-02` isn't available for you. It's available for me on `aarch64-apple-darwin`. Could you show me the error message that you see?

Reply:
(If our macos build server doesn't catch stuff like this then we need to address it.)

Reply:
Maybe some missing `rustup update` on my end, will try again.

Reply:
I'm having this error: `error: the 'cargo' binary, normally provided by the 'cargo' component, is not applicable to the 'nightly-2022-11-02-aarch64-apple-darwin' toolchain`.

Reply:
Did you managed to run a `cargo build` on your side?

Reply:
Ok, I have fix it with `rustup component add cargo`.

Reply:
Hmm, maybe we should add `cargo` to the list of components in `rust-toolchain.toml`

Reply:
Let's leave this for now. I will investigate if it's really needed on another pristine computer.